content/en/users/notebooks/data/_index.md

```diff
@@ -6,35 +6,82 @@ description: >
 weight: 10
 ---
 
-1GB persistent storage for the notebooks is available as your home
-directory. **Please note that files stored on any other folder will be
-lost when your notebook server is closed (which can happen if there is
-no activity for more than 1 hour!)**.
+Every user of the EGI Notebooks catch-all instance has a 1GB persistent
+home to store any notebooks and associated data. The content of this
+home directory will be kept even if your notebook server is stopped (which
+can happen if there is no activity for more than 1 hour). **Modifications
+to the notebooks environment outside the home directory are not kept**
+(e.g. installation of libraries). if you need those changes to
+persist, let us know via a  [GGUS ticket to the Notebooks Support
+Unit](https://ggus.eu). You can also ask for increasing the 1GB home
+via ticket.
 
-If you need to increase your persistent storage space, open a [GGUS
-ticket to the Notebooks Support Unit](https://ggus.eu)
+## Getting data in/out
 
-Access to other kinds of persistent storage for community specific
-instances that can be tailored to your specific needs and available
-storage systems.
+Your notebooks have outgoing internet connectivity so you can
+connect to any external service to bring data in for analysis.
+As with input data, you can connect to any external service to deposit
+the notebooks output.
 
-## Getting your data in
+This is convenient for smaller datasets but not practical for
+larger ones, for those cases we can offer integration with
+several data services. These are not enabled in the catch-all
+instance but can be made available on demand.
 
-Your notebooks have full outgoing internet connectivity so you can
-connect to any external service to bring data in for analysis. We are
-evaluating integration with EOSC-hub services for facilitating the
-access to input data, with EGI DataHub as first target. Please contact
-<support@egi.eu> if you are interested in other I/O integrations.
+### EGI DataHub
 
-## Deposit output data
+[DataHub](../../datahub) provides a scalable distributed data
+infrastructre. It offers a tight integration with Jupyter and
+notebooks with specific drivers that make the DataHub Spaces
+accessible from any notebook.
 
-As with input data, you can connect to any external service to deposit
-the notebooks output.
+The folders are browseable from the notebooks interface. Opening
+files from your code requires you to use the
+[`fs-onedatafs`](https://github.com/onedata/fs-onedatafs) library:
+
+``` python
+from fs.onedatafs import OnedataFS
+
+# create the OnedataFS driver
+odfs = OnedataFS(os.environ['ONEPROVIDER_HOST'],
+                 os.environ['ONECLIENT_ACCESS_TOKEN'],
+                 force_direct_io=True)
+
+# use it to open a file
+f = odfs.open("<datahub file path>")
+```
+
+The `ONEPROVIDER_HOST` and `ONECLIENT_ACCESS_TOKEN` variables
```

Comment:
I guess we can also add that if the user knows the Oneprovider host to connect to, it can use that one instead of the default one

Reply:
ok, added some extra.